packages/core/src/actions/order/_create.ts

```diff
@@ -18,9 +24,10 @@ import { getOrderHashFromOrderV1 } from "../../utils";
  */
 const createOrder = async (
```

Comment:
we might use one object for createOrder params instead of multiple params, allowing to build params dynamically and omit params keys easely.